### PR TITLE
[Improvement-13491][*] Use lombok @Slf4j annotation to generate logger

### LINKED ISSUE #13491: [Improvement][Style] Use lombok @Log annotation to generate loggers
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

* DS uses several `lombok` annotations such as `@data` to make code look neat. We could also use `lombok` `@Log`, `@Slf4j` etc. to generate logger.
* https://projectlombok.org/features/log

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by seedscoder**
## Purpose of the pull request

close  #13491

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

File: lombok.config [hunk 1]
```diff
@@ -16,3 +16,5 @@
 #
 
 lombok.addLombokGeneratedAnnotation = true
+
+lombok.log.fieldname=LOGGER
```
Comment (ruanwenjun) [inline lombok.config]: Why we don't keep the default field name `log`?

Comment (seedscoder) [reply lombok.config]: Lombok will generate a `static final ` field, so I changed the field name to uppercase. Is this inappropriate? Need to change to lowercase？
```java
@Generated
private static final Logger LOGGER = LoggerFactory.getLogger(Xxx.class);
```

> Why we don't keep the default field name `log`?

File: lombok.config [hunk 2]
```diff
@@ -16,3 +16,5 @@
 #
 
 lombok.addLombokGeneratedAnnotation = true
```
Comment (EricGao888) [inline lombok.config]: I prefer that we keep the same formatting of this file. You might either remove the blanks around `=` in this line or add blanks around `=` in next line.

File: lombok.config (hunk 1 above)
Comment (ruanwenjun) [reply lombok.config]: I just think keep the default field name is better, since we already use `@Slf4j` in some class, this will reduce the change.

**Review by ruanwenjun — APPROVED** [review]
LGTM

**Pull request merged**